app/services/user_profiles_encryptor.rb

```diff
@@ -0,0 +1,28 @@
+class UserProfilesEncryptor
+  attr_reader :personal_key
+
+  def initialize(user, user_session, password)
```

Comment:
while we're here, WDYT about a keyword args refactor?

Reply:
Not a bad idea. I'll take care of that real quick